code/modules/reagents/newchem/toxins.dm

```diff
@@ -555,6 +555,40 @@ datum/reagent/tabun/on_mob_life(var/mob/living/M as mob)
 	..()
 	return
 
+/datum/reagent/chloralhydrate
+	name = "Chloral Hydrate"
+	id = "chloralhydrate"
+	description = "A powerful sedative, Lethal in larger doses."
+	reagent_state = LIQUID
+	color = "#000067" // rgb: 0, 0, 103
+
+	on_mob_life(var/mob/living/M as mob)
+		if(!M) M = holder.my_atom
+		if(!data) data = 1
+		data++
+		switch(data)
+			if(1)
```

Comment:
Data will *never* be one. It starts at 0, is then set to 1, and is then incremented, starting data at 2.